src/parse.c

```diff
@@ -2592,3 +2592,19 @@ netplan_get_global_backend()
 {
     return backend_global;
 }
+
+/**
+ * Clear NetplanNetDefinition hashtable
+ */
+guint
+netplan_clear_netdefs()
+{
+    guint n = 0;
+    if(netdefs) {
+        n = g_hash_table_size(netdefs);
+        /* FIXME: make sure that any dynamically allocated netdef data is freed */
```

Comment:
Good catch! I think this might be quite important at some moment, so I'd recommend us creating a ticket for it in the backlog not to forget!

Reply:
Done. As FR-786